app/config.js

```diff
@@ -5,16 +5,16 @@ module.exports = {
 
 	serviceDiscovery: {
 		'gql-schema-registry-db': {
-			host: 'gql-schema-registry-db',
-			port: '3306',
-			username: 'root',
-			secret: 'root',
+			host: process.env.DB_HOST || 'gql-schema-registry-db',
+			port: process.env.DB_PORT || '3306',
+			username: process.env.DB_USERNAME || 'root',
+			secret: process.env.DB_SECRET || 'root',
 		},
 		'gql-schema-registry-redis': {
-			host: 'gql-schema-registry-redis',
-			port: '6379',
-			username: '',
-			secret: '',
+			host: process.env.REDIS_HOST || 'gql-schema-registry-redis',
+			port: process.env.REDIS_PORT || '6379',
+			username: process.env.REDIS_USERNAME || '',
```

Comment:
I think we can drop username and secret for redis, as its not used actually in `app/redis/index.js`.. I added it only to be consistent with config of DB config / service discovery above

Reply:
Although there's no username/password for the Redis docker instance, Redis can be configured to require authentication. Keeping those configuration parameters would support such deployments IMO.

Reply:
hmm.. ok, I'll hope when someone needs it, he will fix passing this

Reply:
I just noticed it wasn't passed. I just modified this code as well in `app/redis/index.js` to account for it. Looking at the `node_redis` documentation, it looks like it only supports passing a password, not a username. Should the username configuration parameter be removed then?

Reply:
doesn't make sense to have useless env vars to me

Reply:
I just removed it.